cpp/src/rolling/rolling.cu

```diff
@@ -287,6 +287,8 @@ struct rolling_window_launcher
   }
 
   // This launch is only for fixed width columns with valid aggregation option
+  // MIN, MAX and COUNT are avilable to all fixed-width types, but SUM and MEAN are valid to
```

Comment:
Mean *is* valid for timestamps!  Sum is not. Here's a summary of what I think should work:

numeric: All, including UDFs
timestamp: min, max, count, mean
string: min, max, count

Also typo (however the whole comment needs rewording).
```suggestion
  // MIN, MAX and COUNT are available to all fixed-width types, but SUM and MEAN are valid to
```

Reply:
@jrhemstad As per our last discussion mean was not supported for timestamp. Does timestamp now have duration type to support mean operation?

Reply:
Timestamp support added